contents/blog/mykonos-hackathon.md

```diff
@@ -0,0 +1,153 @@
+
+Enter the VIP Customer Lookup bot. Ping this clever little Slackbot with any customer ID and it’ll return key details of any account, including the plan they are on, how much money they spend, and who on customer success looks after the account.
+
+### CLI
+
+<BuiltBy people={["Manoel Aranda Neto", "Daniel Esneider"]} />
+
+We build products for engineers, so there’s nothing better than bringing PostHog closer to their natural environment: the terminal.
+
+The PostHog CLI is a command line that allows users to do a few things that are normally tucked away in the PostHog app: creating, reading, updating, deleting, and enabling or disabling feature flags, for example. The PostHog CLI authentication flow is also seamless as it spins up a new browser and allows you to log in with your SSO instead of copying and pasting tokens manually.
+
+In the future, the CLI could be expanded with more features such as creating surveys, and events, installing SDKs automatically, uploading debug symbols, using the CLI as a package, or even as a GitHub action. And, even better, you could do all of that with natural language, no need to memorize all commands by heart.
+
+### 4 years at PostHog
+
+<BuiltBy people={["Coua Phang", "Kendal Hall"]} />
+
+We’re still a young company, but some of our wonderful team members have now been with us for four years or more. So we want to celebrate them. Coua and Kendal came up with a great anniversary gift scheme, meaning our longest-serving colleagues get something special to celebrate. 
+
+This year, Marius, Eric, James G, Lottie, Charles, and Michael all celebrate four years at PostHog and will get to pick between a fancy luggage set (handy for traveling to all-team offsites and other PostHog meet-ups) or a James Hawkins-approved coffee machine.
+
+### Forbidden secret project
+
+<BuiltBy people={["Lior Neu-ner", "Annika Schmid", "Mine Kansu"]} />
+
+If a spinning globe of PostHog events doesn’t convince you we can sell to big enterprises, then our top-secret hackathon project absolutely will. If we ever ship it. Lior, Annika, and Mine use some generative AI magic to create some inspirational quotes from our fearless leader, James Hawkins. 
+
+Click on James’ face on our homepage and he’ll deliver some inspirational zingers that he may or may not have never actually said: “Success isn’t just about getting it right, it’s about crushing every misstep. Embrace the grind, celebrate the wins, and conquer the challenges. Challenges are just opportunities in disguise. Turn setbacks into comebacks and keep pushing forward.” Quick, someone get this on LinkedIn.
```

Comment:
Are we shipping this? Cool if we are, but not sure we should include if not as it doesn't make much sense to anyone else unless they can try it for themselves.

Reply:
OMG I forgot about this. I'll put up a PR to ship and get Eli/Cory to approve. Btw the change was to add it to our [careers page](https://posthog.com/careers), not the home page

Reply:
https://github.com/PostHog/posthog.com/pull/8467

Reply:
Thanks!